packages/ajax/index.js

```diff
@@ -1,22 +1,8 @@
-import { Ajax } from './src/Ajax.js';
-
 export { Ajax } from './src/Ajax.js';
 export { AjaxFetchError } from './src/AjaxFetchError.js';
-
-export let ajax = new Ajax(); // eslint-disable-line import/no-mutable-exports
```

Comment:
Maybe:
```js
export const ajax = new Ajax();
```

So people still have this easy way of making sure they always import the same instance, since ES modules are singletons in essence. Optionally, they can import the Ajax (manager/class) instead if they want to create their own instance, but the most common use case is probably that people will just import the singleton instance.

Reply:
done